crates/cli/src/utils.rs

```diff
@@ -169,6 +169,16 @@ fn file_too_large(file_content: &String) -> bool {
   file_content.len() > MAX_FILE_SIZE && file_content.lines().count() > MAX_LINE_COUNT
 }
 
+// https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
```

Comment:
Can we add a comment here like "use raw ansi escape code to render links in terminal."

Reply:
OK. Updated